src/Features/VisualBasic/Portable/Completion/VisualBasicCompletionOptions.vb

```diff
@@ -0,0 +1,11 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Completion
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Completion
+    Friend Class VisualBasicCompletionOptions
+        Public Const FeatureName As String = "VisualBasic Completion"
+
+        Public Shared AddNewLineOnEnterAfterFullyTypedWord As Options.Option(Of Integer) = New Options.Option(Of Integer)(FeatureName, "Add New Line On Enter After Fully Typed Word", defaultValue:=EnterKeyRule.Always)
```

Comment:
The advantage is that we can now use the same option across langauges (including TypeScript).  We can then move the chekcing of that option to a more base layer (which we probably should be doing as well), and all Roslyn languages will pick up support for it for free.